velox/exec/tests/FunctionSignatureBuilderTest.cpp

```diff
@@ -177,3 +177,30 @@ TEST_F(FunctionSignatureBuilderTest, anyInReturn) {
       },
       "Type 'Any' cannot appear in return type");
 }
+
+TEST_F(FunctionSignatureBuilderTest, constFalgs) {
+  auto signature = FunctionSignatureBuilder()
+                       .returnType("bigint")
+                       .argumentType("double")
+                       .constantArgumentType("boolean")
+                       .argumentType("bigint")
+                       .build();
+  EXPECT_FALSE(signature->constantArguments().at(0));
+  EXPECT_TRUE(signature->constantArguments().at(1));
+  EXPECT_FALSE(signature->constantArguments().at(2));
+  EXPECT_EQ(
+      "(double,constant boolean,bigint) -> bigint", signature->toString());
```

Comment:
Nice test. Would you also add tests for constant variadic argument and constant generic argument?